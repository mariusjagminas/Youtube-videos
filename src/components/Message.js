import React from "react";
import PropTypes from "prop-types";
import Typography from "@material-ui/core/Typography";

const Message = ({ message }) => {
  return (
    <>
      {message ? (
        <Typography component="h3" variant="h3">
          {message}
        </Typography>
      ) : null}
    </>
  );
};

export default Message;

Message.propTypes = {
  message: PropTypes.string
};
