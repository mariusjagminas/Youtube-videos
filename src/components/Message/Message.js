import React from "react";
import PropTypes from "prop-types";
import Typography from "@material-ui/core/Typography";
import Box from "@material-ui/core/Box";

const Message = ({ message }) => {
  if (!message) return null;
  return (
    <Box p={2}>
      <Typography component="h2" variant="h3">
        {message}
      </Typography>
    </Box>
  );
};

export default Message;

Message.propTypes = {
  message: PropTypes.string
};
