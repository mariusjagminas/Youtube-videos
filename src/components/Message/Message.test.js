import React from "react";
import Message from "./Message";
import renderer from "react-test-renderer";

describe("Message component", () => {
  it("match snapshot then it's redered", () => {
    const tree = renderer.create(<Message message={"test message"} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
