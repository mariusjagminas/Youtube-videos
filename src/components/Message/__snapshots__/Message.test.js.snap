// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message component match snapshot then it is not  1`] = `null`;

exports[`Message component match snapshot then it is redered 1`] = `
<div
  className="MuiBox-root MuiBox-root-2"
>
  <h2
    className="MuiTypography-root MuiTypography-h3"
  >
    test message
  </h2>
</div>
`;

exports[`Message component match snapshot then it's redered 1`] = `
<div
  className="MuiBox-root MuiBox-root-2"
>
  <h2
    className="MuiTypography-root MuiTypography-h3"
  >
    test message
  </h2>
</div>
`;
